integration-tests/src/test/java/oracle/weblogic/kubernetes/ItUsabilityOperatorHelmChart.java

```diff
@@ -482,10 +482,10 @@ public void testCreateSecondOperatorUsingSameOperatorNsNegativeInstall() {
       assertNull(opHelmParam2,
           "FAILURE: Helm installs operator in the same namespace as first operator installed ");
     } finally {
-      deleteSecret("ocir-secret",opNamespace);
-      cleanUpSA(opNamespace);
       uninstallOperator(opHelmParams);
       uninstallOperator(op2HelmParams);
+      deleteSecret("ocir-secret",opNamespace);
```

Comment:
use REPO_SECRET_NAME in TestConstants for ocir-secret through the file

Reply:
done